dex/bip-id.go

```diff
@@ -479,7 +479,7 @@ var bipIDs = map[uint32]string{
 	890:   "xsel",
 	900:   "lmo",
 	916:   "meta",
-	966:   "matic",
+	966:   "polygon", // changed from "matic"
```

Comment:
I wonder if we should keep it here as "matic", since our `bipIDs` maps ID to symbol, not coin name.

This npm package addressed it by mapping to both symbol and coin name.  https://github.com/bitcoinjs/bip44-constants#readme
Although I suppose it doesn't really matter if we just use the correct fields from `UnitInfo`...

Reply:
The problems we're seeing are all places where we were using symbol and we should have been using conventional unit. I like this change particularly because it brings those mistakes to the surface.